app/controllers/scim_rails/scim_groups_controller.rb

```diff
@@ -21,21 +21,49 @@ def index
       json_scim_group_response(object: groups, counts: counts)
     end
 
-    # TODO: complete method
     def create
-      group = @company.public_send(ScimRails.config.scim_group_scope).create!(permitted_group_params)
+      group_attributes = permitted_group_params(params)
 
-      json_scim_response(object: group, status: :created)
+      if ScimRails.config.scim_group_prevent_update_on_create
+        group = @company.public_send(ScimRails.config.scim_groups_scope).create!(group_attributes.except(:members))
+      else
+        username_key = ScimRails.config.queryable_group_attributes[:userName]
+        find_by_username = Hash.new
```

Comment:
🌵 any reason you're using `Hash.new` rather than `{}`?
Ruby best practice is to just use `{}`